File: pset1/mario/more/mario.c
#include <cs50.h>
#include <stdio.h>

void printSpace(int nTimes);
void printHash(int nTimes);

// Assignment pset1/mario/more
int main(void)
{
    // Get input from user
    int height;
    do
    {
        printf("Height: ");
        height = get_int();

    }
    while (height < 0 || height > 23);

    // Send spaces and hashes to print function
    for (int i = 2; i <= height+1; i++)
    {
        printSpace((height-i)+1);
        printHash(i);

        // Generate 2 half pyramids with 2 spaces between
        printf("  ");
        printHash(i);
        // printSpace((height-i)+1);
        printf("\n");
    }

}


void printSpace(int nTimes)
{
    // Print number of spaces
    for (int i = 0; i < nTimes; i++)
    {
        printf(" ");
    }
}

void printHash(int nTimes)
{
    // Print number of hashes
    for (int i = 0; i < nTimes; i++)
    {
        printf("#");
    }
}

File: pset6/mario/less/mario.py
from cs50 import get_int

def printHash(nTimes):
    print("#" * nTimes, end='')

def printSpace(nTimes):
    print(" " * nTimes, end='')

height=-1
while height < 0 or height > 23:
    height = get_int("Height: ")

for i in range(2, height+2):
    printSpace(height-i+1)
    printHash(i)
    print()

# print()



